package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "jquery-focuspoint",
+  "version": "1.1.3",
+  "description": "jQuery plugin for 'responsive cropping'. Dynamically crop images to fill available space without cutting out the image's subject. Great for full-screen images.",
+  "main": "js/jquery.focuspoint.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/jonom/jquery-focuspoint.git"
+  },
+  "keywords": [
+    "\"crop\"",
```

Comment:
As I said in first review and @jdreesen pointed out as well, you should not be adding these extra quotation marks. `"\"crop\""` should just be `"crop"` for example.